pyproject.toml

```diff
@@ -54,7 +54,6 @@ python-vxi11 = [
 tests = [
     "pytest>=3.3.0",
     "pytest-cov>=4.1.0",
-    "pytest-qt>=2.4.0",
```

Comment:
A bit cheeky to remove this. I suppose the more correct thing might be to install `PyQt`, but I also figured we could limit our tests to `tests/instruments/`.